src/DssDirectDepositAaveDai.sol

```diff
@@ -144,7 +144,7 @@ contract DssDirectDepositAaveDai {
     event Cull();
     event Uncull();
 
-    constructor(address chainlog_, bytes32 ilk_, address pool_, address _rewardsClaimer, uint256 tau_) public {
+    constructor(address chainlog_, bytes32 ilk_, address pool_, address _rewardsClaimer) public {
```

Comment:
good point! I added that to line 126 since I felt that was a little less buried than in the constructor.  However, I am a little uncomfortable with it there since it implies more permanence than I want for this value (unlike `live` which should pretty much always be `1`). What do you think? 
1) leave it, its fine with the comment
2) move it to the constructor (`tau = 7 days` on line 169)
3) put it back in the constructor params, but default it to 7 days there

Reply:
This is subjective but I think they wasy you did it now is good (so option 1:)).